### PR TITLE
Dp 1186

### PR DESCRIPTION
**Pull request by JordanBalesBAO**
### Related Issues
- DP-XXX
- etc.

## Reviewer:
- [ ] Tests added/updated & passing.
- [ ] Clean linting.
- [ ] Related issue ticket in Jira/GitHub.
- [ ] Documentation added/updated.
- [ ] Code conforms to style guidelines.
- [ ] Well commented.
- [ ] Updates reflected in NEWS.md.
- [ ] Build check passes.

### CODE REVIEW

File: R/writePSNUxIM.R [hunk 1]
```diff
@@ -133,107 +133,17 @@ writePSNUxIM <- function(d,
...
 
     d$info$has_psnuxim <- !is.null(d$data$SNUxIM)
 
     dp_datim_map <- getMapDataPack_DATIM_DEs_COCs(cop_year = d$info$cop_year)
     targets_data <- prepareTargetsData(d, append)
-    template_file <- system.file("extdata", "COP23_PSNUxIM_Template.xlsx", package = "datapackr")
-
+    template_file <- system.file("extdata",
+                                 paste0("COP",
+                                        d$info$cop_year %% 100,
```
Comment (jason-p-pickering) [inline R/writePSNUxIM.R:144]: huh?

File: R/writePSNUxIM.R [hunk 2]
```diff
@@ -334,6 +249,7 @@ writePSNUxIM <- function(d,
     tool <- switch(as.character(d$info$cop_year),
                    "2022" = "OPU Data Pack",
```
Comment (jason-p-pickering) [inline R/writePSNUxIM.R:250]: should be removed i think?

File: R/writePSNUxIM.R [hunk 3]
```diff
@@ -294,7 +204,12 @@ writePSNUxIM <- function(d,
       dplyr::filter(!is.na(org_type)) %>%
       dplyr::select(dp_label, orgUnit = uid)
 
-    schema <- cop23_psnuxim_schema
+    schema <- if (d$info$cop_year == 2023) {
```
Comment (jason-p-pickering) [inline R/writePSNUxIM.R:207]: I think we do have a pickSchema function?

**Review by jason-p-pickering — CHANGES_REQUESTED** [review]
Few small issues which I think can be improved upon

File: R/writePSNUxIM.R (hunk 1 above)
Comment (JordanBalesBAO) [reply R/writePSNUxIM.R:144]: grabs the last two digits of cop year. Ex 2024 will be 24

**Pull request closed without merging**